pymc3/sampling.py

```diff
@@ -265,20 +265,26 @@ def _choose_backend(trace, chain, shortcuts=None, **kwds):
     except KeyError:
         raise ValueError('Argument `trace` is invalid.')
 
+def _make_parallel(arg, njobs):
+    if not shape(arg):
```

Comment:
Does `shape(arg)` really return `False`? Generally it's better to test explicitly.